docs/EN/resources/FAQ.md

```diff
@@ -0,0 +1,195 @@
+<br>
+
+## Omnipod FAQs
+
+### What pods work with Trio?
+You can use DASH and Eros Omnipod pods with the Loop app. You cannot use Omnipod 5 pods. 
+
+### Do I still need a PDM with Trio?
+You do not need the Omnipod Personal Diabetes Manager (PDM); you need the pod supplies. They will pair with only one device at a time. So, a pod is either paired with a PDM or your Trio app on your iPhone. 
+
+In other words, your PDM can stay in the diabetes closet while you are Looping. You cannot use the PDM for a pod activated with the Trio app. That doesn't mean you should get rid of your PDM if you have one. Instead, keep it for backup situations if you lose your phone. 
+
+### Does Trio cause an increase in pod failures?
+There is more communication between your phone and the pod than the PDM. This increases the load on the pod battery. Most people have no increase in pod failures. However, it does occasionally happen. Below are some tips to take to limit battery drain:
+* Have a correction range instead of a single number
+* The pod state is updated every 3 minutes for DASH and every 5 minutes for Eros. If uncertain communications are detected such that the Loop app cannot determine if a dosing command sent to the pod was received, the app will try once a minute to get a response.
+* If using Eros pods with a RileyLink device. Be mindful of situations that may lead to poor signal. Poor communication between devices may lead to battery drain and contribute to faults.
+<br>
+
+## CGM FAQs
+
+### What CGMs are supported by Trio?
+The following CGM can be used with Trio:
+* Dexcom G5
+* Dexcom G6/ONE
+* Dexcom G7/ONE+
+* Medtronic Enlite (must use compatible pump)
+* Libre (limited Models)
+>* US Libre 1 10-day sensors via Transmitters
+>* US Libre 1 14-day sensors via Transmitters
+>* International Libre 1 sensors via Transmitters
+>* European Libre 2 sensors via Transmitters
+>* European Libre 2 sensors directly via Bluetooth
+
+###What CGM apps can be used with Trio?
+**xDrip**
+<br>To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../operate/build.md#xdrip4ios-or-glucose-direct-as-cgm-source).  
+:::{admonition} Tips 
+_If you use xDrip with the Dexcom G6/ONE, in Trio select 'Dexcom G6' as the CGM source. Trio will intercept glucose readings directly from the transmitter._
+:::
+
+**Nightscout**
+<br>Yes. Nightscout can be used as a remote CGM source in Trio. However, this option should be avoided because it does not keep Trio running in the background like other CGM options. You will have to open Trio manually to make it run loop cycles.  
```

Comment:
I believe this has been fixed in Trio 0.2.0's Nightscout integration update, but need @MikePlante1 to confirm.

Reply:
I have left this in, for now; the information is accurate per the current Trio docs. If you are happy to merge, I can update it in a future PR along with the source of the information.